Description: Avoid libutil.h on non-Linux
 Upstream assumes that non-Linux lacks pty.h, but in fact it is
 non-glibc systems that lack pty.h.
Author: "Aaron M. Ucko" <ucko@debian.org>
Reviewed-by: Sean Whitton <spwhitton@spwhitton.name>
Bug-Debian: https://bugs.debian.org/862227
Forwarded: https://github.com/merijn/posix-pty/pull/13
---

--- haskell-posix-pty-0.2.1.1.orig/cbits/fork_exec_with_pty.c
+++ haskell-posix-pty-0.2.1.1/cbits/fork_exec_with_pty.c
@@ -13,9 +13,9 @@
 
 #if defined(__APPLE__)
 #include <util.h>
-#elif defined(__linux__)
+#elif defined(__GLIBC__)
 #include <pty.h>
-#else /* bsd */
+#else /* bsd without glibc */
 #include <libutil.h>
 #endif
 
